godot_project/editor/rendering/rendering.gd

```diff
@@ -220,6 +224,13 @@ func is_renderer_for_motor_built(in_motor: NanoVirtualMotor) -> bool:
 	return true
 
 
+func is_renderer_for_particle_emitter_build(in_emitter: NanoParticleEmitter) -> bool:
```

Comment:
nit: this should be `built` with a t at the end, not a d

Reply:
ah, will fix it in a latter PR, i dont want to rebase a lot of work only for this